src/ert/scheduler/scheduler.py

```diff
@@ -124,18 +125,29 @@ async def run_done_callback(self, state: RealizationState) -> Optional[LoadStatu
             f"Running done callback for {state.realization.run_arg.iens} ..(partly blocking)..",
             end="",
         )
-        callback_status, state._callback_status_msg = await forward_model_ok(
-            state.realization.run_arg
-        )
+        callback_status, status_msg = await forward_model_ok(state.realization.run_arg)
         print(" done")
+
+        if state._callback_status_msg != "":
+            state._callback_status_msg = status_msg
+        else:
+            state._callback_status_msg += f"\nstatus from done callback: {status_msg}"
+
         if callback_status == LoadStatus.LOAD_SUCCESSFUL:
             state.validate()
         elif callback_status == LoadStatus.TIME_MAP_FAILURE:
             state.invalidate()
         else:  # LoadStatus.LOAD_FAILURE
             state.loadfailed()
+
         return callback_status
 
+    async def run_timeout_callback(self, realization: RealizationState) -> None:
```

Comment:
We should unify calling this state and not realization I think.

Reply:
fixed